Build/Projects/MonoGame.Framework.definition

```diff
@@ -146,8 +146,40 @@
   </Services>
 
   <Files>
-
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Properties\AssemblyInfo.Android.cs">
+		<Platforms>Android</Platforms>
```

Comment:
Ups. That's from editing with notepad++ instead of Visual Studio, will fix tomorrow